ftw/testbrowser/core.py

```diff
@@ -316,7 +317,10 @@ def webdav(self, method, url_or_object=None, data=None, view=None,
         """
         self._verify_setup()
         url = self._normalize_url(url_or_object, view=view)
-        driver = self.get_driver(LIB_REQUESTS)
+        driver = self.get_driver()
+        if not driver.WEBDAV_SUPPORT:
```

Comment:
It should not be changed and has therefore a constant character. But it is not stored on module level but on the class for convenient access. I'm not sure what the conventions are here.
Maybe it should be an interface 🤔